mitodl/ocw-to-hugo
# add uid to course.json and course_legacy.json

#### Pre-Flight checklist


#### What are the relevant tickets?
https://github.com/mitodl/ocw-studio/issues/870

#### What's this PR do?
In order to be able to import course uids to ocw-studio, we need to add the uid to course_legacy.json
 
#### How should this be manually tested?
Run
```
node . -i private/input -o private/output -c course_json_examples/example_courses.json --download
```
Go to the /data subfolder for any course in your output.

Verify that `uid` is set in course_legacy.json and course.json
